.travis.yml

```diff
@@ -1,23 +1,24 @@
 language: python
 python:
- - '2.6'
```

Comment:
`2.6` is still listed in the README. Should leave it here if it still works.

Reply:
Travis isn't able to find an archive of 2.6... not sure if that means we should drop support or make this version optional?

Reply:
https://travis-ci.community/t/issue-with-python-2-6-on-linux/3861
> If you need 2.6 support, you need to stick with Trusty.

I say just drop it from here and README.